client/src/components/welcome/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment } from 'react';
+import { Grid, Row, Col, Button } from '@freecodecamp/react-bootstrap';
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+
+import { Spacer } from '../helpers';
+import { randomQuote } from '../../utils/get-words';
+
+import './welcome.css';
+
+function Welcome({ name }) {
+  const { quote, author } = randomQuote();
+  return (
+    <Fragment>
+      <Helmet>
+        <title>Welcome | freeCodeCamp.org</title>
+      </Helmet>
+      <main>
+        <Grid>
+          <Row>
+            <Col sm={10} smOffset={1} xs={12}>
+              <Spacer />
+              <h1 className='big-heading'>Welcome {name ? name : 'Camper'}!</h1>
+            </Col>
+          </Row>
+          <Spacer />
+          <Row className='quote-partial'>
+            <Col sm={10} smOffset={1} xs={12}>
+              <blockquote className='blockquote'>
+                <span>
+                  <q>{quote}</q>
+                  <footer className='quote-author blockquote-footer'>
+                    <cite>{author}</cite>
+                  </footer>
+                </span>
+              </blockquote>
+            </Col>
+          </Row>
+          <Row>
+            <Col sm={10} smOffset={1} xs={12}>
+              <Spacer />
+              <h2 className='medium-heading'>
+                What would you like to do today?
+              </h2>
+            </Col>
+          </Row>
+          <Spacer />
+          <Row>
+            <Col sm={10} smOffset={1} xs={12}>
+              <Button
+                block={true}
+                bsSize='lg'
+                bsStyle='primary'
+                href={`/learn`}
```

Comment:
For internal links use a `Link` helper component or `navigate` function from Gatsby.